extensions/pom.xml

```diff
@@ -64,12 +64,11 @@
                         <id>copy-runtime-dependencies</id>
                         <phase>prepare-package</phase>
                         <goals>
-                            <goal>unpack-dependencies</goal>
+                            <goal>copy-dependencies</goal>
                         </goals>
                         <configuration>
                             <includeScope>runtime</includeScope>
                             <outputDirectory>${project.build.directory}/classes</outputDirectory>
-                            <excludes>META-INF/*.SF,META-INF/*.DSA</excludes>
```

Comment:
Sorry, one other question - is this going to cause any issues with signed JARs? I seem to recall that BouncyCastle is a signed JAR, and some of these items had to be removed to get extensions that use it (RADIUS, maybe?) to run correctly.

Reply:
I believe those issues should no longer occur, since those signed .jar files can now be correctly verified by the JVM.